code/style.css

```diff
@@ -2,6 +2,307 @@
 in this case it is the div with the answer class next to the div with the active class.
 this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
 
-.active + .answer {
-  /* some styling here */
+
+*{
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
+	list-style: none;
+	font-family: 'Montserrat', sans-serif;
+}
+
+/* =================
+header
+==================== */
+body, html {
+  height: 100%;
+}
+
+h1 {
+
+font-family: 'Rock Salt', cursive;
+}
+
+h2 {
+font-family: 'Comforter', cursive;
+font-size: 30px;
+}
+
+.hero-image {
+height: 50vh;
+background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url("./hero-img.jpg");
+height: 50%;
+background-position: center;
+background-repeat: no-repeat;
+background-size: cover;
+position: relative;
+}
+
+.hero-text {
+text-align: center;
+position: absolute;
+top: 50%;
+left: 50%;
+transform: translate(-50%, -50%);
+color: white;
+}
+
+.section{
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+section {
+  padding: 6rem, 2rem;
+	background: #D3DEDC;
```

Comment:
Also a small thing, I think that the padding and background should be on the same line, as you have done in lines 51 - 53, that looks like good indentation and spacing according to https://www.notion.so/technigo/Guidelines-for-how-to-write-good-code-53afa96544e4414b8945132122f030c7